Makefile

```diff
@@ -33,7 +33,7 @@ PATTERN := ^([0-9]|[1-9][0-9]*)\.([0-9]|[1-9][0-9]*)\.([0-9]|[1-9][0-9]*)$
 IS_SEMANTIC_VERSION = $(shell [[ $(or $(BUNDLE_VERSION),$(VERSION),'undefined') =~ $(PATTERN) ]] && echo true || echo false)
 
 IMAGES = submariner-operator submariner-operator-index
-PRELOAD_IMAGES := $(IMAGES) submariner-gateway submariner-route-agent lighthouse-agent lighthouse-coredns
+PRELOAD_IMAGES := $(IMAGES) submariner-gateway submariner-globalnet submariner-route-agent lighthouse-agent lighthouse-coredns
```

Comment:
This is a harmless change, but even without this change, I can deploy (`make deploy using=globalnet`) KIND clusters with globalnet and they seem to work fine.

CC: @skitt @mkolesnik

Reply:
If I run `make prune-images` from any of the repos, it will wipe all Submariner related images. Then go back and build what I want and run:
```
make clusters
subctl deploy-broker --kubeconfig /home/$USER/src/submariner-io/submariner-operator/output/kubeconfigs/kind-config-cluster1 --repository localhost:5000 --version local --globalnet
subctl join --kubeconfig /home/$USER/src/submariner-io/submariner-operator/output/kubeconfigs/kind-config-cluster1 broker-info.subm --clusterid cluster1 --natt=false --repository localhost:5000 --version local
subctl join --kubeconfig /home/$USER/src/submariner-io/submariner-operator/output/kubeconfigs/kind-config-cluster2 broker-info.subm --clusterid cluster2 --natt=false --repository localhost:5000 --version local
```
It works. But when I ping pong back and forth between the `make clusters` and `make deploy`, some of the images are in localhost:5000 (which is done by `make deploy`) and globalnet is not and it hits this state.

Reply:
Okay, thanks for the details @Billy99